src/store/modules/my_profile.js

```diff
@@ -33,6 +36,9 @@ export default {
     },
     setRelayData ({ commit }, relayData) {
       commit('setRelayData', relayData)
+    },
```

Comment:
This seems to have snuck in from another changeset?

Reply:
done